.github/workflows/nightly_release.yml

```diff
@@ -0,0 +1,93 @@
+name: Nightly Builds
+on:
+  schedule:
+    # Runs every weekday at 1am
+    - cron: 0 1 * * 1-5
+  workflow_dispatch:
+
+
+jobs:
+  tag_nightly_release:
+    name: Tag a new nightly release
+    runs-on: ubuntu-latest
+    outputs:
+      tag_name: ${{ steps.set_tagname.outputs.tag }}
+
+    steps:
+      - name: Checkout git repository 🕝
+        uses: actions/checkout@v3
+
+      - name: Compose tag name
+        id: set_tagname
+        run: |
+          DATE=$(date +'%Y%m%d')
+
+          # Find latest rasa-oss version
+          echo "Trying to find the latest rasa-oss version..."
+          LATEST_RASA_VERSION=$(python -c "import sys; import os; sys.path.append('src/rasa/rasa'); from version import __version__; print(__version__)")
+          echo "Current RASA version: ${LATEST_RASA_VERSION}"
+
+          LATEST_NIGHTLY_VERSION=$(echo ${LATEST_RASA_VERSION} | awk -F. '{$(NF-1) = $(NF-1) + 1;} {$(NF) = 0;} 1' | sed 's/ /./g')
+
+          echo "New nightly release version: ${LATEST_NIGHTLY_VERSION}"
+
+          echo "Composing nightly build tag name..."
+          GH_TAG=${LATEST_NIGHTLY_VERSION}-dev${DATE}
+          echo ${GH_TAG}
+          echo "::set-output name=tag::${GH_TAG}"
+
+      - name: Tag latest main commit as nightly
+        uses: actions/github-script@v5
+        with:
+          github-token: ${{ secrets.RASABOT_GITHUB_TOKEN }}
+          script: |
+            github.rest.git.createRef({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              ref: "refs/tags/${{ steps.set_tagname.outputs.tag }}",
+              sha: context.sha
+            })
```

Comment:
won't pushing the tag trigger the [`continuous-integration` workflow](https://github.com/RasaHQ/rasa/blob/44e85366b6410a9b420e12f08b2babd149b10aa9/.github/workflows/continous-integration.yml#L7-L8), especially [the docker image push and pypi deploy](https://github.com/RasaHQ/rasa/blob/main/.github/workflows/continous-integration.yml#L692-L851)?

Reply:
(it's possible to avoid the recursion of workflow when using [the `GITHUB_TOKEN`](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#using-the-github_token-in-a-workflow), but I'm not sure it'll have the necessary permissions)

Reply:
We're about to change that so tags do _not_ trigger the CI workflow, as on tags, we'll trigger the release workflows instead, which will contain the CI workflows.

Reply:
I didnot understand this, I am using the here, 
` with:
          github-token: ${{ secrets.RASABOT_GITHUB_TOKEN }}`